content/_changelogs/10.0.0.md

```diff
@@ -0,0 +1,331 @@
+## 10.0.0
+
+_Released 6/1/2022_
+
+**Summary:**
+
+Component testing is now beta! We've also reworked the Cypress App from the
+ground up to modernize the interface, streamline workflows and integrate better
+into your overall development experience. Read more about 10.0 in
+[our blog post](https://cypress.io/blog/2022/06/01/cypress-10-release/).
+
+**Breaking Changes:**
+
+**<Icon name="exclamation-triangle" color="red"></Icon> Please run
+`cypress open` to go through our interactive migration which will guide you in
+updating your files and configuration options. Read our
+[Migration Guide](/guides/references/migration-guide) which explains some
+breaking changes in more detail.**
+
+- We introduced several breaking changes to the **Cypress configuration file**
+  detailed below:
+  - Using a `cypress.json` configuration file is no longer supported. Replace
+    this configuration file with a `cypress.config.js`, `cypress.config.ts`,
+    `cypress.config.cjs` or `cypress.config.mjs` file. This change provides
+    better flexibility and extensibility of configuration. An error will be
+    thrown if both a `cypress.json` file and `cypress.config.{cjs,js,mjs,ts}`
+    file are found. Addressed in
+    [#18221](https://github.com/cypress-io/cypress/pull/18221),
+    [#18300](https://github.com/cypress-io/cypress/pull/18300),
+    [#20554](https://github.com/cypress-io/cypress/pull/20554), and
+    [#20643](https://github.com/cypress-io/cypress/pull/20643).
+  - A Cypress configuration file is now required and passing the CLI flag
+    `--config-file=false` will throw an error. Addressed in
+    [#20643](https://github.com/cypress-io/cypress/pull/20643).
+  - A default configuration file will no longer be auto-generated by Cypress
+    when a configuration file doesn't exist. Instead, you can run `cypress open`
+    and set up your project with a specific testing type to have this file
+    generated. This allows you to customize the configuration setup by
+    testing-type for the language and dependencies leveraged in your project.
+  - The configuration file's JSON schema is no longer supported since the
+    configuration can no longer be provided via a JSON format.
+- Cypress has made several **configuration option changes** including renaming
+  options, removing options, and only allowing some options within a specific
+  testing type. Please run `cypress open` to have our automated migration update
+  these options for you.
+
+  - Many configuration options are now specific to either end-to-end or
+    component testing. The types have also been updated accordingly to match.
```

Comment:
Style nit: "accordingly" and "to match" have the same meaning here, we can pick one. Maybe a slight tweak like this would make it more clear as somebody skims through:

```suggestion
    component testing. The types have also been updated to match the new structure.
```